### ISSUE TITLE
Users unable to enter new patients on platform (urgent)

### ISSUE THREAD

**Issue by SAbdin93** [rcpch/rcpch-audit-engine#1116]
Hi Michael & Simon, 

I wanted to flag an urgent issue on the platform. Users are currently unable to enter new patients from Cohort 6, when they try to input a first paediatric assessment date, they receive an error message. I've attached a screenshot of the error message (Edit @mbarton - removed, contact me for the error message)

The dashboard shows that we are in Cohort 7 (I think because of the cohort dates - and yesterday was the 1st December) however it should still show as Cohort 6 until the data entry deadline in January 14th. 

Let me know if there is anything you need from me, happy to meet over Teams if needed.

Thank you! 
Saleema

**Comment by mbarton** [comment]
Thanks @SAbdin93, I am now looking into this. I have put up a banner on all pages on the site indicating issues:

> We are investigating issues entering new patient data, please check again later

Thank you for the screenshot however I have removed it for now as it contained a real patient name and this is a public GitHub page

**Comment by SAbdin93** [comment]
@mbarton Thanks Michael. Sorry to be a pain but can the message say the following: 

We are currently experiencing issues with entering new patient data. In the meantime, users can still edit records for children and young people already on the system. Thank you for your patience as we work to resolve this.

**Comment by mbarton** [comment]
I am able to reproduce this now in staging and locally in my dev instance. Note that superusers or the audit team override this check automatically here:

https://github.com/rcpch/rcpch-audit-engine/blob/d4f201366f912405044bc6f676eef2cfb0ee1974/epilepsy12/views/registration_views.py#L603

which confused me why I couldn't reproduce the problem

**Comment by mbarton** [comment]
The hotfix is now live and I have removed the banners on the site indicating issues. @SAbdin93 and I have tested it is working in staging (so therefore will work in live as they run the exact same code).

The hotfix entirely removes the validation check for first paediatric assessment date. This was the fastest way to restore service to our users. We will put the check back correctly handling the submission grace period in #1117 

I have updated the banner on the organisation dashboard landing page giving reassurance that cohort 6 submission is still open pending discussion about how to change the wording on the page itself, as we are technically now in cohort 7 but with 6 still open for submission.

Closing this incident as resolved
